apps/electric_cli/lib/electric_cli/options.ex

```diff
@@ -20,6 +20,16 @@ defmodule ElectricCli.Options do
     @default_flags
   end
 
+  def local_flags do
+    [
+      local: [
+        long: "--local",
+        help: "Disable authentication check for commands executed against the local console.",
+        required: false
```

Comment:
This flag is not required but you're matching against `flags: %{local: local}` in commands. Doe Optimus always add flags to the `flags` map with the default value of `false` if they are not passed to the command?

Reply:
Yes it does.